andela/pippin-ah-frontend
# [Feature: #164012830] Display Articles on the Frontend

#### What does this PR do?
- This `PR` adds the feature for displaying articles on the frontend

#### Description of Task to be completed?
- articles should display on the frontend

#### How should this be manually tested?
- navigate to `articles/:category`
- you should see the articles in that category

#### What are the relevant pivotal tracker stories?
- [#164012830](https://www.pivotaltracker.com/story/show/164012830)

<img width="1434" alt="screen shot 2019-02-23 at 10 56 45 pm" src="https://user-images.githubusercontent.com/43746609/53292129-62a7f880-37be-11e9-8fbc-17e60291eb8c.png">
